### ISSUE TITLE
Please add some more templates

### ISSUE THREAD

**Issue by Awais-cb** [AmruthPillai/Reactive-Resume#1428]
### Is there an existing issue for this?

- [X] Yes, I have searched the existing issues and it doesn't exist

### Feature Description

Please add some more templates or if there's any option to add any new template please suggest

**Comment by b2111** [comment]
I agree, and ideally, more flexibility in customizing the existing ones.

**Comment by AmruthPillai** [comment]
More templates are definitely in my list of things to do. But I'm curious to know what other ways of customization for a template you would like to see?

**Comment by AkashKumar7902** [comment]
Hey @AmruthPillai 
here are some customization: 
https://drive.google.com/file/d/17_OhUkgyxY3MGMj4jyEzwKhvYTaVv_pW/view?usp=drive_link
https://drive.google.com/file/d/18wTvaVLgy15Eji9AxDy0Ti0PtYsCGSOx/view

**Comment by AmruthPillai** [comment]
@AkashKumar7902 This is extremely helpful, especially since I'm building a whole new templating system and plan to add a lot more templates like these in the new version.

Will look into implementing these designs into the app :) Thank you!

**Comment by AmruthPillai** [comment]
This issue was automatically closed as it was created before 2024-01-01.
